pkg/queue/cluster_queue.go

```diff
@@ -95,7 +95,7 @@ func newClusterQueue(cq *kueue.ClusterQueue, wo workload.Ordering) (*ClusterQueu
 func newClusterQueueImpl(wo workload.Ordering, clock clock.Clock) *ClusterQueue {
 	lessFunc := queueOrderingFunc(wo)
 	return &ClusterQueue{
-		heap:                   heap.New(workloadKey, lessFunc),
+		heap:                   *heap.New(workloadKey, lessFunc),
```

Comment:
Is it a bugfix after the previous PR, or this is specifically needed for this? Or not needed ?:)

Reply:
Not needed, but I want to stay consistent and always have a `New_` function return a pointer.

Reply:
Sure, consistency is important